execution_workflows/LABRAT/modules/salmon_quant.nf

```diff
@@ -7,7 +7,9 @@ def options    = initOptions(params.options)
 process SALMON_QUANT {
         tag "$sample"
         publishDir "${params.outdir}/labrat/salmon/", mode: params.publish_dir_mode
-        container "quay.io/biocontainers/salmon:0.14.0--h86b0361_1"
+        container "quay.io/biocontainers/salmon:1.6.0--h84f40af_0"
+
+        cpus 8
```

Comment:
I'm not familiar with salmon but is 8 cpus the standard for this process or is it a good idea to parameterize this in the config file so the user can adjust the number if they don't want to use the default 8 cpus for this process?

Reply:
My understanding is, that this number is treated as the maximum. If you supply less cpus somewhere in the config or on the CLI, it would only use those. 
So I think salmon does not profit from more than 8 cpus, however this is subjective. 
Again, I can remove this statement. The one below with `${task.cpus}` should still work though.